src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/database_tests/ReportRepositoryTests/ReportTests.kt

```diff
@@ -334,7 +334,7 @@ class ReportTests : CleanDatabaseTests()
     @Test
     fun `can get latest dates for selected reports`()
     {
-        val now = Instant.now()
+        val now = Instant.ofEpochMilli(1655378424228)
```

Comment:
`Instant.now()` gives greater precision that the datetimes coming out of the db, so create an Instant that has millisecond precision.